templates/download/shared/_server_weekly_news.html

```diff
@@ -1,83 +1,59 @@
-            </li>
-            <li><small>All information provided will be handled in accordance with the Canonical <a href="/legal">privacy policy</a>.</small></li>
-            <li class="p-list__item">
-                <div class="g-recaptcha" data-sitekey="{{ CAPTCHA_TESTING_API_KEY }}" style="margin: 2rem 0;"></div>
-            </li>
-            <li class="mktField p-list__item">
-              <button type="submit" class="mktoButton p-button--positive">Subscribe and download</button>
-              <input type="hidden" name="formid" class="mktoField" value="3485">
-              <input type="hidden" name="formVid" class="mktoField" value="3485">
-              <input type="hidden" name="munchkinId" class="mktoField" value="066-EOV-335">
-              <input type="hidden" name="Consent_to_Processing__c" value="yes" />
-              <input type="hidden" name="thankyoumessage" value="Thank you for subscribing!<br><a href='https://assets.ubuntu.com/v1/f401c3f4-Ubuntu_Server_CLI_pro_tips_2020-04.pdf' target='_blank' >Here is your CLI cheat sheet.</a>" aria-lable="">
-            </li>
-          </ul>
-      </fieldset>
-      </form>
-      <div class="js-feedback-notification"></div>
-    </div>
+      <input class="mktoField" value="yes" id="CanonicalUpdatesOptIn" name="canonicalUpdatesOptIn" type="checkbox">
+      <label class="mktoLabel mktoHasWidth" for="CanonicalUpdatesOptIn"><small>I would like to receive occasional news from Canonical by email.</small></label>
+      <p><small>All information provided will be handled in accordance with the Canonical <a href="/legal">privacy policy</a>.</small></p>
+      <div class="g-recaptcha" data-sitekey="{{ CAPTCHA_TESTING_API_KEY }}" style="margin: 2rem 0;"></div>
+      <button type="submit" class="mktoButton p-button--positive">Download the cheat sheet</button>
+      <input type="hidden" name="formid" class="mktoField" value="3485">
+      <input type="hidden" name="formVid" class="mktoField" value="3485">
+      <input type="hidden" name="munchkinId" class="mktoField" value="066-EOV-335">
+      <input type="hidden" name="Consent_to_Processing__c" value="yes" />
+      <input type="hidden" name="returnURL" value="https://assets.ubuntu.com/v1/f401c3f4-Ubuntu_Server_CLI_pro_tips_2020-04.pdf" />
+    </form>
   </div>
-</section>
+</div>
+
+<script>
+  /**
+    Toggles visibility of modal dialog.
+    @param {HTMLElement} modal Modal dialog to show or hide.
+  */
+  function toggleModal(modal) {
+    if (modal && modal.classList.contains('p-modal')) {
+      modal.classList.toggle('u-hide');
+    }
+  }
+
+  // Add click handler for clicks on elements with aria-controls
+  document.addEventListener('click', function (event) {
+    event.preventDefault();
+    var targetControls = event.target.getAttribute('aria-controls');
+    if (targetControls) {
+      toggleModal(document.getElementById(targetControls));
+    }
```

Comment:
@anthonydillon this was a bit of a head melt to work out! This is stopping the submit event from being fired on the form (and also from checking the email checkbox in the form) - without this line, it submits as expected.

```suggestion
    var targetControls = event.target.getAttribute('aria-controls');
    if (targetControls) {
      event.preventDefault();
      toggleModal(document.getElementById(targetControls));
    }
```

Reply:
Awesome, thanks. Good find!

Reply:
I assumed it was the form-validation script but was looking in the wrong place.